lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
```

Comment:
I answered to that suggestion in your previous comment. The problem is that we can have the user as the operation performer, the user being changed or both. So we can't bind a single user object.